scserver.js

```diff
@@ -639,6 +639,9 @@ SCServer.prototype.generateId = function () {
 };
 
 SCServer.prototype.addMiddleware = function (type, middleware) {
+  if (!this._middleware[type]) {
+    throw new InvalidArgumentsError(`Middleware type "${type}" is not supported. \nRead more: https://socketcluster.io/#!/docs/middleware-and-authorization`)
```

Comment:
Avoid `\n` new lines and references to websites inside error messages (keep them simple), but feel free to add references to the website inside comments above the error message if you like.

Reply:
Also missing semicolon at the end of the line.